Configuration/FormConfigurationFactory.php

```diff
@@ -345,10 +345,6 @@ private function getEmailFromDynamic(Dynamic $dynamic)
      */
     private function getEmail($email, $name = null)
     {
-        if (!$email || !$name) {
```

Comment:
the `!$email` check should still exist else the fallback mechanism for from and to doesn't work.

```
        if (!$email) {
            return null;
        }
```

Reply:
@codebarista did correct the check